contents/blog/clickhouse-vs-redshift.md

```diff
@@ -0,0 +1,150 @@
+
+## Why ClickHouse is still faster (with a caveat)
+
+The general consensus of speed tests is that ClickHouse is faster [[1](https://clickhouse.com/blog/redshift-vs-clickhouse-comparison)][[2](https://www.firebolt.io/comparison/redshift-vs-clickhouse)][[3](https://altinity.com/blog/clickhouse-vs-redshift-performance-for-fintech-risk-management)]. However, this speed comparison isn’t as cut-and-dry as most other ClickHouse OLAP comparisons. 
+
+The trickiness of the ClickHouse and Redshift comparison is that it comes down to the nitty-gritty. Both ClickHouse and Redshift offer materialized views with incremental updates, SIMD vector executions, and strong compression standards. The difference is that Redshift’s optimizations are created to optimize queries without much configuration (and, by contrast, some limited efficacy), while ClickHouse’s optimizations take work and effort, but are tuned for a project’s unique needs. 
+
+There are some limitations of Redshift that ClickHouse [calls out](https://clickhouse.com/blog/redshift-vs-clickhouse-comparison) itself. The first limitation is that Redshift only allows up to 50 concurrent queries; this is a valid concern, particularly for analytics or data products where concurrent queries would be abundant. 
+
+The second limitation is that Redshift needs to compile first-time queries to optimize them, which adds a few seconds of delay. This compilation time isn’t ideal for analytics products where users are constantly customizing first-time queries to the database. ClickHouse also notes that it has better compression than Redshift, scoring something in the realm of 2x compression.
+
+A common criticism of ClickHouse, meanwhile, is that it lacks a results cache, which is a flagship feature of Redshift. A results cache is excellent for applications with predictable, repeatable queries. Thankfully, ClickHouse has addressed this concern with a new update in February of this year; the project now supports [a configurable cache](https://clickhouse.com/blog/introduction-to-the-clickhouse-query-cache-and-design). 
+
+A counterargument to ClickHouse’s speed supremacy is that it requires a lot more work. Materialized views need to be configured by engineers. ClickHouse, in general, requires more technical oversight to streamline queries. 
+
+Meanwhile, Redshift has AutoMV, which automatically creates materialized views. Redshift is ideal for companies that want a faster solution than Google BigQuery, but cannot afford to spend too much time tuning queries. 
+
+## How ClickHouse and Redshift often work together
+
+It’s common for ClickHouse and Redshift to work together. Because ClickHouse is faster at certain queries, while Redshift is a more general-purpose product that is easier to connect to a preexisting AWS stack, they can work well together with redundant data. ClickHouse [has a fantastic guide](https://clickhouse.com/blog/redshift-vs-clickhouse-comparison) detailing step-by-step how to connect the two. 
```

Comment:
```suggestion
Because ClickHouse is faster at certain queries, while Redshift is a more general-purpose product that is easier to connect to a preexisting AWS stack, they can work well together with redundant data. ClickHouse [has a fantastic guide](https://clickhouse.com/blog/redshift-vs-clickhouse-comparison) detailing step-by-step how to connect the two. 
```